app/src/main/java/com/vultisig/wallet/ui/models/VaultAccountsViewModel.kt

```diff
@@ -100,6 +100,14 @@ internal class VaultAccountsViewModel @Inject constructor(
         }
     }
 
+    private fun showVerifyFastVaultPasswordReminderIfRequired(vaultId: VaultId) {
+        viewModelScope.launch {
+            if (vaultMetadataRepo.isFastVaultPasswordReminderRequired(vaultId)) {
```

Comment:
@yvebe , we should only pop up the dialog when the vault is a fastvault , don't popup the dialog when it is secure vault